eng/Localize/LocProject.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Projects": [
+    {
+      "LanguageSet": "Azure_Languages",
+      "LocItems": [
+        {
+          "SourceFile": "src\\polyglot-notebooks-vscode-insiders\\package.nls.json",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\polyglot-notebooks-vscode-insiders"
+        },
+        {
+          "SourceFile": "src\\dotnet-interactive\\Resources.resx",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\dotnet-interactive"
+        },
+        {
+          "SourceFile": "src\\Microsoft.DotNet.Interactive\\Resources.resx",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\Microsoft.DotNet.Interactive"
+        },
+        {
+          "SourceFile": "src\\Microsoft.DotNet.Interactive.AspNetCore\\Resources.resx",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\Microsoft.DotNet.Interactive.AspNetCore"
+        },
+        {
+          "SourceFile": "src\\Microsoft.DotNet.Interactive.Journey\\Resources.resx",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\Microsoft.DotNet.Interactive.Journey"
+        },
+        {
+          "SourceFile": "src\\Microsoft.DotNet.Interactive.ExtensionLab\\Resources.resx",
```

Comment:
Please remove this project from the loc work.

Reply:
removed